main/provider/index.js

```diff
@@ -241,6 +241,7 @@ class Provider extends EventEmitter {
 
   approveRequest (req, cb) {
     log.info('approveRequest', req)
+    accounts.lockRequest(req.handlerId)
```

Comment:
is there anywhere else we may want to lock the request? I know right now we don't keep them around but we might want to in the future. not sure if it matters, just an idea

Reply:
Yeah, possibly on decline, though it doesn't matter at the moment as we delete declined requests